### DISCUSSION TITLE
Add flap speeds to PFD

### DISCUSSION THREAD

**Discussion by fafaldo** [magknight/787aviatorsTracker#688] [Ideas]
Would you be able to add flap speed bugs to PFD's speed tape? I know doing it 'properly' would require a custom FMS to be implemented, but maybe just add a simplified versions? For example ones that would use calculations based on VREF tables found in aircraft manuals (depending on flap settings, gross weight etc) + simple VREF+10, VREF+20, VREF+30 (etc)?

**Comment by aeroniemi** [comment]
We already have V1/VR/V2/VREF/VREF+5 - those are the core ones that require calculations behind them; all others are just fixed speeds. The issue with adding them is working out the conditions to actually show them, which might actually be possible now with flightState. VREF+10/20/30 aren't shown in the 787, at least not under any of the configurations that I've seen.
